src/tools/dev_server/dev-server.ts

```diff
@@ -60,8 +60,9 @@ async function launch() {
   hotReloadServer.start();
 
   console.log(`${green(bold('ALDS Started.'))}\n\nWeb server port: ${port}\nExplorer port: ${explorePort}\nHotReload port: ${hotReloadPort}`);
-  console.log(`\n${green('Next, try visiting:')}`);
-  console.log(`- An Arcs shell (web-shell, dev-shell, or pipes-shell)`);
+  console.log(`\n${green('Next, try visiting an Arcs shell at:')}`);
+  const shells = ['web-shell', 'dev-shell', 'pipes-shell'];
+  shells.map(shell => console.log(`- http://localhost:${port}/shells/${shell}`));
```

Comment:
I didn't include this originally because there is no mapping to Google3. 
Do you have an idea of how to handle the internal case?

Reply:
I think we should just add an instruction for g3 users that is relevant to them (when we have it). Additionally, my understanding was that they would be able to use the dev and web shells for prototyping.